src/main_helpers.js

```diff
@@ -33,6 +33,7 @@ export function findInvestBinaries(isDevMode) {
       // If no dotenv vars are set, default to where this project's
       // build process places the binaries.
       serverExe = `${process.env.SERVER || 'build/invest/server'}${ext}`;
+      //serverExe = `${process.env.SERVER || 'build/invest/server'}`;
```

Comment:
I left this commented code here, because when I was running in `dev` mode using `.env` and `server.py` the auto adding of `ext` was causing issues. Should the resolving line be:
```javascript
serverExe = `${process.env.SERVER}` || `build/invest/server${ext}`;
```

Reply:
Think I have it updated here: https://github.com/natcap/invest-workbench/pull/73/commits/b592fc7592d3d61a45d152369c75b2e25d92b10c